data/strategies/StrategyMakerUNIDAIDelegate.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyMakerUNIDAIDelegate",
+  "description": "Locks UNI at MakerDAO via UNI-A to mint DAI. This newly minted DAI is then deposited into the v2 DAI yVault.",
```

Comment:
Do we want to say something about the debt ratio? or may tiny bit longer explanation of this strat?